packages/monaco/src/browser/monaco-keycode-map.ts

```diff
@@ -0,0 +1,155 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as browser from "@theia/core/lib/browser";
+
+const KeyCode = monaco.KeyCode;
+
+export let KEY_CODE_MAP: { [keyCode: number]: monaco.KeyCode } = {};
```

Comment:
it seems to contain the same info  as `MONACO_KEY_CODE_MAP`:
https://github.com/theia-ide/theia/blob/45884b21389ecf421b8d9cdfa2a708fb608faab5/packages/monaco/src/browser/monaco-keybinding.ts#L18-L155

Reply:
Oh yes, I didn't see that. It's the other direction though. I would like to keep my copy (with proper copyrights) and remove this one with a function that iterates.